csi.proto

```diff
@@ -634,6 +829,25 @@ message NodeGetIdResponse {
   // CO in subsequent `ControllerPublishVolume`.
   // This is a REQUIRED field.
   string node_id = 1;
+
+  // Specifies where (regions, zones, racks, etc.) the node is
+  // accessible from.
+  // A plugin that returns this field must also set the
+  // ACCESSIBILITY_CONSTRAINTS plugin capability.
+  // COs may use this information along with the topology information
+  // returned in CreateVolumeResponse to ensure that a given volume is
+  // accessible from a given node when scheduling workloads.
+  // This field is OPTIONAL. If it is not specified, the CO MAY assume
+  // the node is not subject to any topological constraint, and MAY
+  // schedule workloads that reference any volume V, such that there are
+  // no topological constraints declared for V.
+  //
+  // Example 1:
+  //   accessible_topology =
+  //     {"region": "R1", "zone": "R2"}
+  // Indicates the node exists within the "region" "R1" and the "zone"
+  // "Z2".
+  Topology accessible_topology = 2;
```

Comment:
Change the call to `GetNodeInfo()` instead?

Reply:
we are really trying to stop making breaking changes with new releases of CSI.  I don't think there's a good way to rename message types in protobuf without breaking backward compatibility, so to me the rename doesn't seem worth it.

If we find other reasons to make breaking changes, and 0.3 becomes incompatible with 0.2, then I agree that GetNodeInfo() would be better, but we could do that in a separate PR later.

Reply:
+1. I think we can introduce a new `NodeGetInfo` call (the response contains both id and other information about the node) and deprecate `NodeGetId`.